doc/randomNumberGeneration.md

```diff
@@ -0,0 +1,73 @@
+# Random number generation
+
+Random number generation can be set from the Gaudi options file, including setting an array of seeds:
```

Comment:
This first part should really be dropped now, and only the UniqueIDGenSvc should be advertised.
And probably this should go to k4FWCore

Reply:
Yeah I agree, I will close this PR and open one there with only the UniqueIDGenSvc part.